src/main/resources/changelog-master.xml

```diff
@@ -128,4 +128,6 @@
     relativeToChangelogFile="true"/>
   <include file="changesets/changelog-consent-2023-07-27-match-rationale.xml"
     relativeToChangelogFile="true"/>
+  <include file="changesets/changelog-consent-2023-09-20-capitalize-datasets-used-in-user-role-dao.xml"
+    relativeToChangelogFile="true"/>
```

Comment:
nit: I think this name is misleading, all we're really doing here is renaming `roleid` -> `role_id`

Reply:
Done!